src/db_parser.jl

```diff
@@ -63,545 +353,1075 @@ function map_prime_mover(prime_mover::String)
+                                  get_base_power(p.base_system),
+                available=component_attr["available"],
+                x=transform_natural_impedance_to_device_base(
+                    component_attr["x"],
+                    arc_dict[arc],
+                    p.base_system,
+                ),
+                r=transform_natural_impedance_to_device_base(
+                    component_attr["r"],
+                    arc_dict[arc],
+                    p.base_system,
+                ),
+            )
+        end
+        if haskey(component_attr, "uuid")
+            IS.set_uuid!(IS.get_internal(l), Base.UUID(component_attr["uuid"]))
+        end
+        add_component!(p.base_system, l)
+
+        #If Line is between areas, add to dictionary
+        from_area = PSY.get_name(get_area(get_from(get_arc(l))))
+        to_area = PSY.get_name(get_area(get_to(get_arc(l))))
+        if from_area != to_area
+            areas = Set([from_area, to_area])
+            if haskey(tx_dict, areas)
+                push!(tx_dict[areas], PSY.get_name(l))
+            else
+                tx_dict[areas] = [PSY.get_name(l)]
             end
-        elseif row["fuel_type"] == "Hydro"
-            continue
-        else
-            t = ThermalStandard(;
-                name=row["name"],
+        end
+
+        #Build Portfolio transmission based on existing transmission
+        if get_aggregation(p) == PSY.ACBus
+            component_attr = get(attributes, rec.id, Dict{String, Any}())
+            arc = first(DBInterface.execute(db, QUERIES[:arc], [rec.arc_id]))
+            balancing_topology_from =
+                first(
+                    DBInterface.execute(db, QUERIES[:topology_from_arc], [arc.from_id]),
+                ).name
+            balancing_topology_to =
+                first(
+                    DBInterface.execute(db, QUERIES[:topology_from_arc], [arc.to_id]),
+                ).name
+            t = NodalACTransportTechnology{ACBranch}(;
+                name=rec.name,
+                id=rec.id,
                 available=true,
-                status=true,
-                bus=filter(b -> contains(b.name, row["balancing_topology"]), buses)[1],
-                active_power=row["rating"],
-                reactive_power=0.0,
-                rating=row["rating"],
-                active_power_limits=(0.0, row["rating"]),
-                reactive_power_limits=nothing,
-                ramp_limits=nothing,
-                operation_cost=ThermalGenerationCost(
-                    variable=CostCurve(LinearCurve(variable_om)),
-                    fixed=op_data[!, "fom_cost"][1],
-                    start_up=0.0,
-                    shut_down=0.0,
-                ),
-                base_power=row["base_power"],
-                prime_mover_type=map_prime_mover(row["prime_mover"]),
-                fuel=map_fuel(row["fuel_type"]),
+                power_systems_type=string(ACBranch),
```

Comment:
This topic can be tricky. You get different results if you `using PowerSystems` vs `import PowerSystems`. You can call `string(nameof(ACBranch))`. However, if your type if ever parametric, the parameters will be dropped. We use `IS.strip_module_name` for this in many cases.